helix-core/pom.xml

```diff
@@ -139,6 +139,11 @@ under the License.
       <artifactId>commons-math</artifactId>
       <version>2.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-math3</artifactId>
+      <version>3.0</version>
```

Comment:
If you really decide to use, would it be better to use the latest one 3.6.1?

Reply:
Sure.